From: "Trevor Joynson (trevorj)" <github@skywww.net>
Date: Thu, 7 Jul 2016 02:15:46 -0400
Subject: Bump ver

---
 debian/changelog | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: spl-linux/debian/changelog
===================================================================
--- spl-linux.orig/debian/changelog
+++ spl-linux/debian/changelog
@@ -1,4 +1,4 @@
-spl-linux (0.6.5.7-1ubuntu1) yakkety; urgency=medium
+spl-linux (0.6.5.7-0ubuntu99) yakkety; urgency=medium
 
   * Git pull to support 4.7.0-rc5
 
